src/utils/APIError.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line max-classes-per-file
+const httpStatus = require('http-status');
+
```

Comment:
i think we should less focus on the es6 class semantic as per the data in node 14 super and extend keyword are much slower that functional programming.
http://incaseofstairs.com/six-speed/

Reply:
We can discuss more on this, I wanted to keep our error handling mechanism.